cluster.go

```diff
@@ -47,32 +47,34 @@ import (
 
 type mongoCluster struct {
 	sync.RWMutex
-	serverSynced  sync.Cond
```

Comment:
Hmm, reformat? Are we not using `gofmt`, or do we have `gofmt` fighting with itself?